src/wp-includes/block-editor.php

```diff
@@ -529,6 +612,10 @@ function get_block_editor_settings( array $custom_settings, $block_editor_contex
 		),
 	);
 
+	if ( ! empty( _wp_get_post_content_block_attributes() ) ) {
+		$editor_settings['postContentAttributes'] = _wp_get_post_content_block_attributes();
```

Comment:
`_wp_get_post_content_block_attributes()` call multiple time if condition true instead define a variable and use it.

Reply:
Makes sense!